CONTRIBUTING.md

```diff
@@ -2,53 +2,4 @@
 
 We appreciate that you're interested in helping with moving the project forward. Before you submit your first PR, please read the following guide. We'd hate to see you work on something that someone else is already working on, something that we agreed not to do or something that doesn't match the project.
 
-Sharing is caring!
```

Comment:
I don't believe the aim of this issue was to remove most of the guidance here but rather update it to align with the current approach and add a link to full details. 
@milanholemans as you are the author of the issue then maybe you may confirm. 
in many places the Contribuiting.md is tracked and shown as a pointer to contribution guidance. If we just clear out this file and only leave the link it will only become problematic for someone who is in search of this guidance as now he or she would need to actually first redirect to contribuiting.md and then redirect again to docs.

Reply:
I haven't read the entire contributing page, but I guess most of the things there are still relevant and we should only update links to the GH wiki.